### ISSUE TITLE
[Feature]: Season 1 Launch changes

### ISSUE THREAD

**Issue by kempsterrrr** [mhmulford0/dd-website-monorepo#5]
### Context / Background / How This Idea Came To Fruition

To support the launch of Season 1 I'm proposing the following changes:

- [x] Edit existing, or add new banner, to clearly state message regarding $CODE launch date and link to claim site
- [x] Add Claim Site link to NavBar + Footer
- [x] Maybe add School of Code and Perks Link to NavBar if folks are up for it
- [x] Add link to D_D blog to NavBar (LOL we missed this)
- [ ] Developer-DAO/dd-website-monorepo#11 also needs to be closed before the Season starts.
- [ ] Update intro component and associated CMS home page data model to reflect $CODE launch chagnes

### Problem / Opportunity

When launching a token, OpSec is a real concern and there are going to be people trying to scam folks in the community with fake mint sites, fake tokens and DM scams. We need to do whatever we can on the website to help protect against this.

### Proposed Solution / How It Can Be Improved

See list above

### Testing the changes

Manual

**Comment by kempsterrrr** [comment]
For th first point I think we can probably use the existing marqeue banner, what do you think @mhmulford0 @andriishupta ?? Everything else seems pretty straight forward to me.

**Comment by andriishupta** [comment]
@kempsterrrr I have added the "news_ticker" field to the Home Query response so that we can pass this kind of information.

Currently, it consist from Statement{ title: string; description }[], but we can change it to be Link{ title, url, }[] and pass links to Claim site
This Marquee header is located "above" Home page, so we can create a separate query for General/Settings to fetch on _app.tsx

For Footer same - we can add link to existing Component footer -> useful_links(Link[]) | discover(Link[]) | social(Link[])

**Comment by andriishupta** [comment]
short: answer -> yes, we can use existing Header and pass info from CMS + add required info to Footer

**Comment by kempsterrrr** [comment]
Great shout re the CMS @andriishupta - I wonder if we just get that field for the ticket to be a markdown block so we can put whatever text we want into the banner without having to worry about the data model?

**Comment by andriishupta** [comment]
yes it could be w/e we want

if that Marquee accepts Markdown correctly we can just create RichText field "content"

pseudo
```
query general {
  news_ticker {
    content // rich text
  }
}
```

**Comment by kempsterrrr** [comment]
Looks great to me mate. If you've got the time go for it otherwise I'll pick it up at the weekend :)

**Comment by kempsterrrr** [comment]
Just dropping a link to another issue in the CMS repo re markdown in the CMS - https://github.com/Developer-DAO/cms/issues/12

**Comment by andriishupta** [comment]
I would pick up CMS right away cause there shouldn't be many efforts, but with FE, we should check if we could easily just paste Markdown to Marque component

**Comment by andriishupta** [comment]
after merge of related CMS PR - we could query for { general } in _app.tsx to pass `news_ticker` to Marquee

**Comment by kempsterrrr** [comment]
remove underline from footer navlinks... add blog, school of code and p3rks

**Comment by mulf0** [comment]
I have created this branch to sync all of the Season 1 todos. please use it as the base for any of those items. (also linked under _development_ on the right side)

https://github.com/Developer-DAO/developer-dao-dot-com/tree/5-feature-season-1-launch-changes

**Comment by kempsterrrr** [comment]
Opened #15 to address these changes, see preview link in that PR for latest version.

**Comment by kempsterrrr** [comment]
@andriishupta @mhmulford0 working on merging in the `/terms` and `/privacy-policy` pages to close out #11 

Right now our data model in the CMS has the footer and the header data in the home page type. now we have multiple pages this no longer suits us as far as I can tell. Two suggested solutions:

1. Seperate Footer and Header single types in the CMS
2. Looking deeper into the [Navigation plugin](https://market.strapi.io/plugins/strapi-plugin-navigation) we've spoke about elsewhere

I'm about to sign off for the day but will pick this up again tomorrow. If either of you have opinions on this please share them otherwise 🙏🏻  I'll do my best to sort it out tomorrow and share my solution.

**Comment by andriishupta** [comment]
@kempsterrrr, as a suggestion in chat or in GitHub issues, I mentioned that we could move the footer to General type

as I remember - we don't have a Header on the home page, but the heading / sub_heading - "Build web3 with friends."

suggestions:
- move Footer to General
- create Header(with plugin)
- it makes sense to change the Layout logic

our changes with "Moving Home Page to CMS" didn't introduce logic changes to FE code, so new can do it now

**Comment by kempsterrrr** [comment]
@andriishupta Apologies, by "Header" I'm referring to the global `<Header />` component which includes the navigation.

If we're using the plugin given it can create multiple navigations imho it would be better to just create a both the Header navigation and footer navigation in there so they can be controlled via the cms. what do you think?

**Comment by andriishupta** [comment]
Yes, 100% we should do it, if navigation plugin would be suitable also for footer

**Comment by andriishupta** [comment]
now we can do more with ease when we have both projects at the same place and "unblocked" by using monorepo  🙇
